lib/extensions/conditional/active_record/base.rb

```diff
@@ -20,4 +22,12 @@ def specific_conditions
       conditions.map(&:actable)
     end
   end
+
+  module ConditionInstanceMethods
+    # A human-readable name for each condition; usually just wraps a title
+    # or name field. Meant to be used in a polymorphic manner for views.
+    def name
+      fail 'To be implemented by each condition.'
```

Comment:
I think I told @minhtule to remove that line because it's no different from not defining the method. but there needs to be a place to put this info.

Yes, the most straightforward way is to include this in a class and not override the method.

By the way -- should this raise a NotImplementedError?